stestr/commands/__init__.py

```diff
@@ -1,22 +0,0 @@
-# Licensed under the Apache License, Version 2.0 (the "License"); you may
-# not use this file except in compliance with the License. You may obtain
-# a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
-# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
-# License for the specific language governing permissions and limitations
-# under the License.
-
-from stestr.commands.failing import failing as failing_command
-from stestr.commands.init import init as init_command
-from stestr.commands.last import last as last_command
-from stestr.commands.list import list_command
-from stestr.commands.load import load as load_command
-from stestr.commands.run import run_command
-from stestr.commands.slowest import slowest as slowest_command
-
-__all__ = ['failing_command', 'init_command', 'last_command',
```

Comment:
This is going to break backwards compatibility and external consumers. (like I'm pretty sure I used this in os-testr somewhere) We can't remove this interface to the command functions without a deprecation period.

Reply:
Oh, I got it. I see the point.
https://github.com/openstack/os-testr/blob/master/os_testr/ostestr.py#L156